src/PhpSpreadsheet/Chart/Title.php

```diff
@@ -87,4 +95,23 @@ public function getLayout()
     {
         return $this->layout;
     }
+    /**
+     * Get font
+     *
+     * @return Font
+     */
+    public function getFont() {
+        return $this->font;
+    }
+
+    /**
+     * Set font
+     *
+     * @param Font $font
+     * @return Title
+     */
+    public function setFont(Font $font = null)  {
```

Comment:
If font defaults to null, the property should also allow null; and the writer should also check for null font when reading the colour